.travis.yml

```diff
@@ -57,9 +57,6 @@ jobs:
     - name: 3D smFISH data processing example
       if: type = push and branch =~ /^(master|merge)/
       script: make install-dev 3d_smFISH.py
-    - name: iss_cli.sh data processing example
```

Comment:
You can plausibly remove the Makefile rules for running `.sh` data processing examples.

Reply:
ping

Reply:
removed!